source/documentation/rdo-packaging-guidelines.html.md

```diff
@@ -78,6 +112,37 @@ oslo-config-generator --config-file=config-generator/keystone.conf
 
 * Configuration files must be in /etc and not /usr/etc.
 
+### Tests packaging
+
+OpenStack projects provide different tests including unit tests and functional
+tests,  typically using the tempest framework.
+
+* Core packages shouldn't include tests as are not required in runtime.
+
+* Unit tests should be included in a &lt;package name>-tests-unit package that
+should depend on the test requirements.
+
+* Tempest tests should be included in a &lt;package name>-tests-tempest package
+which should depend on the dependencies to run the provided tests. Note that
+some projects include tempest tests in the main project git repository (so tempest
+package would be a subpackage in the same spec file) while others use a separate
+git repository (so a specific distgit and spec will be needed).
+
+* &lt;package name>-tests: includes all tests, and should be a virtual package
+requiring &lt;package name>-tests-tempest and &lt;package name>-tests-unit.
+
+### Other considerations
+
+* To enforce consistency accross OpenStack services packages, use the following snippet to set upstream project name.
+
+```bash
+%global service keystone
+```
+
+Depending on the package type (OpenStack service, library, client, etc...) There
```

Comment:
I'd just drop the different variables part and emphasize that we provide different example specs to match the different types of projects packaged.